.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: "Release Documentation"
+
+"on":
+  push:
+    branches:
+      - 'develop'
+      - '#5670'
+
+jobs:
+  gen-docs:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Check out vega code
+        uses: actions/checkout@v3
+        with:
+          path: 'vega'
+      -
+        name: "Setup ssh key for PR"
+        uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.DOCUMENTATION_KEY }}
+      -
+        name: Check out docs code
+        uses: actions/checkout@v3
+        with:
+          path: 'documentation'
+          repository: 'vegaprotocol/documentation'
+          ssh-key: ${{ secrets.DOCUMENTATION_KEY }}
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      -
+        name: Get dependencies
+        run: cd vega; bash script/gettools.sh
+      -
+        name: Generate docs
+        run: cd vega; make proto_docs; echo "generated docs:"; find protos/generated -type f
+      -
+        name: Make PR to the docs repo
+        env:
+          GH_TOKEN: ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }}
+        run: |
+          mkdir -p documentation/specs/$GITHUB_REF_NAME
+          cp -r vega/protos/generated/* documentation/specs/$GITHUB_REF_NAME/
+          ls -al documentation/specs/$GITHUB_REF_NAME/
+          cd documentation
+          git config --global user.email "vega-ci-bot@vega.xyz"
+          git config --global user.name "vega-ci-bot"
+          branch_name="$(date +%d-%m-%Y--%H-%M)-spec-update"
```

Comment:
Could this change to 'spec-update-[tag]'? i.e. 'spec-update-v0.54.1'

Reply:
We can do it but I would opt for a timestamp for a reason that we might do few updates to the same tag as we want to trigger this action on push to develop

Reply:
@mkjmdski when we chatted about this I opted for on tag instead - in which case would tag be ok here?